jaxrs/WFLY-11943_Statistics_of_REST_endpoints.adoc

```diff
@@ -0,0 +1,78 @@
+= WFLY-11943 / Statistics of REST endpoints
+:author:            Rebecca Searls
+:email:             rsearls@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Collect and report runtime statistics on REST endpoints running in RESTEasy.
+
+== Issue Metadata
+
+=== Issue
+
+* {issue-base-url}/WFLY-11943[WFLY-11943]
+* {issue-base-url}/EAP7-606[EAP7-606]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFLY-11751[WFLY-11751]
```

Comment:
Could you please explain why https://issues.jboss.org/browse/WFLY-11751 is related to statistics on REST endpoints?

Reply:
Endpoint meterics are displayed under "rest-resource-paths" and this is
an element that is listed under jaxrs/rest-resource.  WFLY-11751 deals
with display of jaxrs/rest-resource information.